app/models/host.rb

```diff
@@ -639,10 +639,13 @@ def verify_credentials_task(userid, auth_type = nil, options = {})
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
 
-  def verify_credentials?(*args)
+  def verify_credentials?(auth_type = nil, options = {})
     # Prevent the connection details, including the password, from being leaked into the logs
     # and MiqQueue by only returning true/false
```

Comment:
Hmm, this comment is no longer true.

We are now putting the new password into the queue. It is the only way I could think of passing the new password to the `ems_operations` worker (who is actually verifying the password)

@agrare are you ok with putting the prospective credentials onto the queue?

Reply:
Yes as long as it is filtered out in the logs as a ManageIQ::Password object (this is the same for the EMS verify_credentials method)

Reply:
I just remembered why we have this comment (was reviewing the `VerifyCredentialsMixin` that `ExtManagementSystem` uses).  It was so that the _results_ weren't leaked into the logs, not the parameters.